src/pudl/extract/excel.py

```diff
@@ -44,6 +44,12 @@ class Metadata:
     * column_map/${page}.csv currently informs us how to translate input column
       names to standardized pudl names for given (partition, input_col_name).
       Relevant page is encoded in the filename.
+
+    Optional file:
+
+    * page_part_map.csv tells us what page is connected to an additional partition
+      outside the partition included in the rest of the mapping files (usually year).
+
```

Comment:
`page` is the terminology used throughout the Excel mapping csvs, which just corresponds to the table/asset that is created. It's called `page` because it typically corresponds to the tab/page of the Excel spreadsheet we're extracting.

Reply:
But I can clarify this description a bit to make it clearer.

Reply:
hmmm yeah that probably makes sense. Ive never used `page` in the context of Excel. Is there a reason we used "page" instead of "tab" in the first place? @cmgosnell

Reply:
the whole excel extractor is designed around extracting "pages". these mostly correspond to "all the data in a tab of an excel file... but i think it is defined more generically on purpose. bc we've used to extract dbf data and are converting it to extract data from csv's. and if there are multiple chunks of data you want to extract from the same tab you could use `skiprows` and `skipfooter` to grab multiple pages from a tab if multiple tables were for instance stacked on top of eachother.  or if they were side by side but different tables you could use a different set of column maps for the same tab. iirc the name "page" was a many years ago jan decision. I'm not sure its a perfectly apt name, but changing it definitely feels oos for this pr.

but still out of the full context of the doc string for even this metadata class the page may sound out of place, but could you read the full docstring for the class and then let us know if it still feels confusing?